docs/workflow/blocks/http.rst

```diff
@@ -62,15 +64,32 @@ If the endpoint needs to be constructed from data, the endpoint can be specified
         }
     }
 
+.. code-block:: json
+
+    {
+    "type": "http",
+    "method": "get",
+    "endpoint": {
+        "protocol": "https:",
+        "host": "api.harvestapp.com/api/v2",
+        "pathname": "/reports/time/tasks",
+        "valueGetters": {
+            "query": "{ from: context.savedData.from, to: context.savedData.to }"
+        }
+    }
+  }
+
 
+**Headers** 
 For advanced use cases, the payload can be constructed using a JMES Path expression.
-Custom headers can also be specified using JMES Path expressions:
+Custom headers can also be specified using JMESPath expressions.
+Be careful to write your header's value with two types of quote (double quotes + single quotes), if the value is a string.
```

Comment:
This part needs an example probably.
I tried to improve it as is though:
```suggestion
Caution: if the custom header value is a string, header values must use two types of quotes: double quotes and single quotes.
```

Reply:
Yeah, you are right, is not clear, but I don't know how to explain it. 
I am not sure about your suggestion, neither, though.
Why did you write `custom header`? This tip (double+single quotes) is valid for any headers, custom or not. Or when we say "custom header" we mean "non-dynamic value"?

There is an example, is right after this text, in the http block example. https://github.com/kendraio/kendraio-app/pull/393/files#diff-544912a428316debac6dc397877aae4d88c8bdac4836974e3caa2c8389ffc96dR98

But yeah, maybe we can be more explicit and add this line `"payload": "'grant_type=client_credentials'"` in the explanation.

Something like:
> Caution: if the header value is a string, the header value must use two types of quotes: double quotes and single quotes like in the following example: `"payload": "'grant_type=client_credentials'"`

What do you think @lukestanley ?

Actually, I have no idea if it should be in double + single quotes in the case the value is a number or boolean. 
But I don't think we will ever have a header that is not either a string or a dynamic value

Reply:
Actually I'd replace this:
`For advanced use cases, the payload can be constructed using a JMES Path expression.
Custom headers can also be specified using JMES Path expressions:
Custom headers can also be specified using JMESPath expressions.`
With:

`JMESPath expressions can be used to dynamically set header values.` Rather than the duplication. This would be more clear.

And thanks for pointing out that there is an example below. @gsambrotta

Reply:
Good catch to avoid repetition. I think it still good to mention the payload tho. 
What about this text for the whole description:
```
JMESPath expressions can be used to dynamically set header and payload values.
Caution: if the header value is a string, it must use two types of quotes: double and single quotes, like "payload": "'grant_type=client_credentials'".
```
Is this good for you @lukestanley ?

Reply:
I thumbed up suggested text last week. So a commit with that in it is needed, right?  @gsambrotta